### PR TITLE
* Implement #1023

### PR DESCRIPTION
**Pull request by Wagnerp**
* Implement #1023 

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs [hunk 1]
```diff
@@ -40,13 +40,13 @@ public KryptonTextBoxActionList(KryptonTextBoxDesigner owner)
         /// <value>The Krypton Context Menu.</value>
         public KryptonContextMenu? KryptonContextMenu
         {
-            get => _textBox.KryptonContextMenu;
+            get => _textBox?.KryptonContextMenu;
 
             set
             {
-                if (_textBox.KryptonContextMenu != value)
+                if (_textBox?.KryptonContextMenu != value)
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs:47]: If text box is null, and value is not then line 51 will fail

File: Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs [hunk 2]
```diff
@@ -58,13 +58,13 @@ public KryptonTextBoxActionList(KryptonTextBoxDesigner owner)
         /// </summary>
         public PaletteMode PaletteMode
         {
-            get => _textBox.PaletteMode;
+            get => _textBox!.PaletteMode;
 
             set
             {
-                if (_textBox.PaletteMode != value)
+                if (_textBox?.PaletteMode != value)
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs:65]: if textbox is null and value is not, then line 68 will fail

**Review by Smurf-IV — CHANGES_REQUESTED** [review]
The comment applies to all changes with the if statement detection, as it seems to be a pattern you have used

File: Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs]: Reverted

File: Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs]: Reverted

**Comment by Wagnerp** [comment]
Reverted changes in `KryptonTextBoxActionList`

**Pull request merged**